doc/classes/InputEventWithModifiers.xml

```diff
@@ -27,6 +27,10 @@
 		<member name="shift" type="bool" setter="set_shift" getter="get_shift" default="false">
 			State of the [kbd]Shift[/kbd] modifier.
 		</member>
+		<member name="store_command" type="bool" setter="set_store_command" getter="is_storing_command" default="true">
+			Serialises [member command] instead of [member control] on Windows, and [member meta] on Mac.
+			This aids with cross-platform compatibility when developing on Windows for OSX, or vice-versa.
```

Comment:
The world is bigger than Windows and macOS (and "Mac" is not a standard name).
Also `en_US` spelling :)

```suggestion
			If [code]true[/code], pressing [kbd]Command[/kbd] on macOS or [kbd]Control[/kbd] on all other platforms will both be serialized as [member command]. If [code]false[/code], those same keys will be serialized as [member meta] on macOS and [member control] on all other platforms.
			This aids with cross-platform compatibility when developing e.g. on Windows for macOS, or vice-versa.
```

Reply:
Edited `kbd` key names to match existing docs.